core/server/data/migration/index.js

```diff
@@ -63,9 +63,9 @@ init = function init(tablesOnly) {
             logger.info('Database initialisation required for version ' + versioning.getDefaultDatabaseVersion());
             return populate(logger, tablesOnly);
         }
-        // 3. The database exists but the currentVersion setting does not or cannot be understood
-        // In this case the setting was missing or there was some other problem
-        errors.logErrorAndExit('There is a problem with the database', err.message);
+        // CASE: the database exists but the currentVersion setting does not or cannot be understood
```

Comment:
FYI - the 3 here is a reference to a comment higher up.... maybe these should be CASE A, CASE B, CASE C etc rather than numbers so they don't look weirdly out of order.